CHANGES/5278.misc

```diff
@@ -0,0 +1 @@
+Adding is_highest filter for Collection Version.
```

Comment:
Since this is an addition that is user facing I think it should be `5278.feature`.

Reply:
Also similar w/ the single quotes, backticks, etc.